target/scripts/helpers/log.sh

```diff
@@ -39,15 +39,23 @@ function _log
     return 1
   fi
 
-  local MESSAGE LEVEL_AS_INT
+  local MESSAGE LEVEL_AS_INT FALLBACK_LOG_LEVEL
```

Comment:
We have `LOG_LEVEL`, that's why I would prefer `LOG_LEVEL_FALLBACK` here instead.

```suggestion
  local MESSAGE LEVEL_AS_INT LOG_LEVEL_FALLBACK
```

Reply:
I can do that :) But I will need to adjust more than just this line. I will need to manually push a commit as I cannot just apply this suggestion :)

Done in 3e5093b3b80d65509c8a672ea253d8f5d220eded

Reply:
> But I will need to adjust more than just this line

Of course. But I was confident that you know that 👍 So I did not suggest n changes.